medusa/metadata/generic.py

```diff
@@ -368,30 +369,22 @@ def create_season_all_banner(self, show_obj):
             return self.save_season_all_banner(show_obj)
         return False
 
-    def _get_episode_thumb_url(self, ep_obj):
+    def _get_episode_thumb_url(self, series, episode):
```

Comment:
Do we really need to pass the series? I believe the series is already available by doing ep.series?

Reply:
Possibly, I will have another look.

Reply:
We need information from the indexer series object, we can't get that from the episode object currently. I called it ```indexer_series``` now to make it more obvious.